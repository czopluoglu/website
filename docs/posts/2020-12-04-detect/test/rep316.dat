10001000100010000010
11110111110100101111
10010000000001111010
01011101001100101111
11111100111111111111
11110000001010101011
10101011001011111110
11111111111001100011
01100100011100011111
10010000001011100111
01010110111010000111
11111111100000101110
10101001100010101100
00100001101101000011
00100101100000101110
11011111101000100110
10100001111111001010
01001000000100001000
10010001001010101011
11000101111001011110
11101100101011001000
10011100001001111111
11011110111101000111
10000101101101110111
00100011101110110111
11110010001101101111
01111001001010000010
11101101111111101001
01111000110011001101
11010000011000100110
00111010110011110100
11100101001001111101
10000000000100011011
11011100111110001111
10001011101110000101
01010011001111011110
10011101000111110111
00111011111111101111
01011001011110001101
10001000001010110000
01110101111011100011
11100111011111111110
11111111111001101111
11111111111000001011
11111111111011001111
11000001101111110110
01111011111010000001
01000001000000001100
10011001001111111111
10000001101110100111
10000001101010101010
11101000010010000011
11100100111110110000
10101111111111100010
00011101001100110011
11011000111001000000
11001001110010000110
10111001010000000100
10001000101010101011
10100111111100101111
10101011110100101111
00001000001100110111
11111100110100110110
00010001111000111110
01100101011101100101
10101101111110101111
11110101111000111010
11010001011011011011
11010011101010011100
01110110100110011101
11001100111111111101
11100100001001100111
01011011011100101111
11001111100111010111
11101110101101111111
10000001101110101001
11011001001110110110
10100011011111111111
01011100110010100010
11110111111010101011
01000101011011111111
01011111110110100110
10101011101011100111
00010001010010000000
00101011111011111101
11111110011111011011
10111101011010110001
10100001100100011111
00000000110111100111
01100001011111101101
01011111011011011011
11001100110010100110
01000101111011011111
11101000100010111011
11111101111101101111
00001101011111111011
10101101001010100001
01001000110100000001
11011110110001100111
00100011101011101111
11010101011110100010
11011100011110101101
11000001100010101011
10000000010000000110
00001101110011110101
10001111001010000011
00011011101111001111
00001001011110000001
11011001111100101110
10111111101010011011
11011101011011111110
11001101110010001010
01111011101111111111
00100100011100111110
11101110111111111111
10111001011111101001
10011011110111011110
10001101001111111010
01111000010101101001
10010101100001100110
00000100000010100011
11001011110111011010
10000000101101010110
00110001010010000111
10111001111010100111
11110110011000011101
01100101111011111101
00101000011110001000
01010000101111011110
10111010011000000010
10101000111101111011
00010001011100100011
11001010100100000010
11101011101110101000
00001100100010000010
00010100111100110011
11110101011110111110
11110010101110111111
01101000101111111011
10100111011001111111
11011111111100101110
11000111011100110011
00001000000000001111
10100001101110010011
10011101011000110110
10101111111010101111
11001001001110000010
00001110100111100111
10011101001110111111
01110110011101111010
10100100100001000111
11001111011110101111
01000000111100011110
00001100011110111110
10111101110010100110
10000000001010010010
10011011010000000100
00101101101000000101
11100001001011111011
01000110101101111111
01101101010010100110
01000000001001100111
00010101000010001011
00110001011010111100
01100001101111001111
11011110101010100111
01010001000011100000
01100111001110111110
00111111011001111100
01001001011000111011
10101001000101001011
00100000000010101101
10010101111011111011
01011001111010110110
10101101100110100010
00000001111010111001
11110101010010001001
11000101111000011111
01000001101000101011
00001011101110110111
10100001001010101111
10100101100000110010
11100001100001100011
11001101001110111111
01110001001100100110
00101110101011100011
11100111011100111011
10010001010010011101
01100101110111001010
11110111111010010011
01111110101010011001
01001000001111110011
10110111111000100010
10110101001011110111
01010000101000011001
10001100001000110110
00000110111101000111
01111001000010001010
11111111110110100111
10110110110111111010
00011101010000110000
11100001011111111010
00000000001100000111
01001001010101101010
11000010101111010111
10001111011000100001
00110101010000101111
01100111101111111011
11001110001000111111
11101010111001001011
01001100001000000111
11011011101111101111
01010000101011010110
11111111011101110111
01111011111110101110
10011001111100100110
11110111011101111110
00000101100000000111
10100000001111111100
01010100001001100011
01110111110101100111
11100000110111010010
10101001111111110010
00101010001111111111
00000000111111000111
00001101110011010110
11110111100110110111
00000000000000100011
00001000000000110010
10011111011001100110
11001000000000001000
01100001111010100111
10001011111111101111
11010101110000000011
11010100101010101111
10010001010110000011
10111100110000001111
11000101111110111001
11000001000010100010
10101101111110101011
01110011111010110010
01110111111110101111
01000001101110001111
00100010101111010110
11011010010000110111
10011000101010111100
11111111111010100011
11010000001101111100
01100000101111111010
00011011101101110111
