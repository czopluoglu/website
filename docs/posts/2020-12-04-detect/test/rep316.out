-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep316.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.64
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7892
                   IDN index value:   0.6579
                           Ratio r:   0.5688

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    1    1    1    3    2    3    1
    4    4    2    4    4    4    4    3    3    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    4    5    6   10

   -----------CLUSTER  2-------------

    3    8   13

   -----------CLUSTER  3-------------

    7    9   18   19   20

   -----------CLUSTER  4-------------

   11   12   14   15   16   17
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d-+++-++--+-+----+--
   -d+-++---++-+-++---+
   ++d+++-+-+-+-----+--
   +-+d++--+++------+--
   ++++d+++-+-+++---+--
   -++++d++------------
   +---++d++++--+--++-+
   +-+-+++d+---+----+--
   ---+--++d-+---+-+-++
   -++++-+--d+++++++---
   ++-+--+-++d++--++---
   --+-+----++d++-++++-
   ++--+--+-+++d--+++--
   ----+-+--+-+-d+++--+
   -+------++---+d+++++
   -+-------++++++d+-++
   ------+-++++++++d-++
   +-+++-++---++-+--d+-
   --------+--+--++++d+
   -+----+-+----++++-+d
 
 No cross validation for this DETECT run
